src/pyOpenMS/tests/unittests/conftest.py

```
@@ -0,0 +1,348 @@
+  """
+  Test helper to validate string output type across Python versions.
+    
+  Args:
+    input_str: String to validate
+        
+  Asserts:
+    - Python 2: input_str is unicode
+    - Python 3: input_str is str
+  """
+  if sys.version_info[0] < 3:
+    assert isinstance(input_str, unicode)
```

Ruff (0.14.10)

Undefined name `unicode`

(F821)